May/detection rules with one setup doc (#31330)
DataDog/documentation: layouts/shortcodes/cloud_siem/job_multi_triggering.md

> In the **Job multi-triggering behavior** section, configure how often you want to keep updating the same signal if new values are detected within a specified time frame. For example, the same signal updates if any new value is detected within 1 hour, for a maximum duration of 24 hours.

Reviewer: ```suggestion
In the **Job multi-triggering behavior** section, configure how often to keep updating the same signal when new values are detected within a specified time frame. For example, the same signal updates when any new value is detected within 1 hour, for a maximum duration of 24 hours.
```

Reviewer: nitpicking. feel free to ignore

> - An `evaluation window` is specified to match when at least one of the cases matches true. This is a sliding window and evaluates cases in real time.

Reviewer: ```suggestion
- An `evaluation window` defines a sliding period in which at least one case evaluates as true, assessing cases in real time.
```

Reviewer: just trying a shorter version. feel free to ignore.
